Jenkinsfile

```diff
@@ -7,11 +12,17 @@ def buildApps(appList) {
   }
 }
 
-def runningOn(machine) {
-  println "Stage running on:"
-  println machine
+def buildDocs(){
+  sh 'pip install git+ssh://git@github.com/xmos/xmosdoc'
+  sh 'xmosdoc html latex'
+  script {
+    def doc_version = sh(script: "cat settings.yml | awk '/version:/ {print \$2}'", returnStdout: true).trim()
+    def zipFileName = "docs_xscope_fileio_v${doc_version}.zip"
```

Comment:
Is it so important to have the version in the zip name? Ultimately the artifact belongs to a build and should only be accessible through a portal which makes it obvious which version it represents. 
Whilst reasonably small, the whole script block could be removed if we remove this requirement.